packages/vscode-plugin/README.md

```diff
@@ -1,6 +1,10 @@
 # Harper for VS Code
 
-Harper is the grammar checker for developers. It checks for spelling and grammar errors in your Markdown files and code comments. You can find out more by checking it out on [GitHub](https://github.com/automattic/harper) or by visiting the [website](https://writewithharper.com).
+Harper is the next-generation grammar checker for your code. It catches common stylistic errors, as well as complex grammatical or layout-related problems. It works for almost [all common programming languages](./language-server#Supported-Languages) and a number of markup formats.
+
+If you use Rust, Java, JavaScript, or any number of other programming languages, you comments may be ending up as part of your API's documentation. If that's the case, grammatical mistakes in your code could be down-ranking your site on search results and tarnishing your reputation for quality.
```

Comment:
You have a typo: **you** comments may be ending up → **your** comments may be ending up